### ISSUE TITLE
submit two riot apps to playstore as a poor man's multiaccount support

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-android#1353]
one for personal account; one for professional account, or similar, until we have proper multiaccount

**Comment by jcgruenhage** [comment]
Another way to tackle this would be to give different builds different packages. 
Example:  

| Build | package |
| ---- | ---- |
| F-Droid Release | im.vector.riot.fdroid.release |
| F-Droid Debug | im.vector.riot.fdroid.debug |
| Play Store Release | im.vector.riot.googleplay.release |
| Play Store Debug | im.vector.riot.googleplay.debug |

Or something like that.

**Comment by pganssle** [comment]
This is awful. If people want to go this route, I would think that they should use [one of the suggestions here](https://www.quora.com/How-do-I-install-the-same-app-multiple-times-on-my-Android-device) to clone the app on their own phone.

Adding a second version to the app store will just confuse people, and probably won't make for a particularly pleasant user experience anyway.

**Comment by maxidor** [comment]
agreed with the horrible user experience. when telling people to just find and install Riot, they won't know which one to install...

**Comment by Valodim** [comment]
I think this is actually a quite beautiful solution! (that's why I suggested it)

Having two apps in the play store covers the use case of connecting to two independent homeservers perfectly, and with minimal development effort. It's a straightforward user experience, the apps are separated in a way that is easy to understand and trust. Multiple accounts in the same app always feel iffy, e.g. if you want to connect to an internal business homeserver.

Patching apps to install them multiple times simply isn't a solution that works for most people.

**Comment by Valodim** [comment]
I can see the argument about confusing users, so here's a fun suggestion: Call the second version "Riot Pro", give it a blue R icon rather than a green one, sell it for 1.99€, and make it very clear in the description that it's the same as the free one :)

Think about it: The target audience for this use case is people who would like to connect to two homeservers, one of them likely a company-hosted one. 1.99€ is pocket change so it won't bother the people who actually have this use case, it brings in a small amount of money, but at the same time it makes it perfectly clear which app to install for typical users. It's even a nice way to support the project with a very low barrier (everyone buys apps, not everyone has paypal), heck I'd go for that blue R.

**Comment by maxidor** [comment]
@Valodim That's actually a very good idea

**Comment by jcgruenhage** [comment]
People don't read descriptions and will assume that the free version is limited. Just an assumption, but still.

**Comment by Mikaela** [comment]
So if I happened to host five different homeservers or had a different account on them all, would you send 5 clones of the same app to Play Store? In my opinion the only good suggestion in this thread is jcgruenhage's first comment (https://github.com/vector-im/riot-android/issues/1353#issuecomment-310279317) which would have a practical purpouse to allow stable and beta/debug to be installed at the same time.

> Call the second version "Riot Pro", give it a blue R icon rather than a green one, sell it for 1.99€, and make it very clear in the description that it's the same as the free one :)

And what if the user has no payment card or desire to give it to Google or just has no money?

I think the only real solution is https://github.com/vector-im/riot-android/issues/120 which is done by Telegram, possibly Slack (I am not sure if it allows the same team to exist multiple times) and majority of XMPP clients.
